packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx

```diff
@@ -161,7 +165,133 @@ const PayablesTableBase = ({
     }
   }, [isError, error, i18n]);
 
-  const gridApiRef = useAutosizeGridColumns(payables);
+  const columns = useMemo<GridColDef[]>(() => {
+    return [
+      {
+        field: 'document_id',
```

Comment:
you removed display: flex for columns 'document_id' and 'created_at'.
It breaks vertical spacing for the payable in OCR status (see screenshot).
Bring back display: flex if possible. Otherwise, please find a way to center icons vertically (probably wrap them with flex)

<img width="904" alt="Screenshot 2024-08-15 at 20 32 35" src="https://github.com/user-attachments/assets/23ff5e14-6c1a-47b7-961b-f10b1608046e">